### DISCUSSION TITLE
Wallet connections: 0

### DISCUSSION THREAD

**Discussion by ghost** [Chia-Network/chia-blockchain#1727] [Q&A]
Me and several others are having issues with the wallet tab in the GUI showing connections as 0. This is just for updating the balance right? And would not have any affect on farming?

I pass filter and have many peer connections. I thought maybe it is because I've yet to farm a block and have no established wallet on the ledger yet? Other who have farmed blocks are having the issue as well though.

![Untitled1](https://user-images.githubusercontent.com/61402138/113778046-e94b9580-96f9-11eb-8865-9891bd885fa3.jpg)

**Comment by hoffmang9** [comment]
The Wallet has no impact on farming other than seeing your farming rewards.

**Reply by ghost** [reply]
Thanks for the confirmation. That was my assumption, but I wanted to verify as others on reddit/discord have mentioned having this issue as well. I will relay this response to them.

**Reply by jelloir** [reply]
@hoffmang9 - thanks for the response to this.  My farm was showing plots passing filters, everything sync'd etc but wallet connections = 0.  I took my farmer offline for some planned maintenance, bought it back online, after which it showed wallet connections = 2.  Within a couple of hours I had won 2 blocks after not winning for a few days.  Probably random right?

**Reply by hoffmang9** [reply]
It would not surprise me that you had won but your wallet had not synced to the winning height yet.

**Reply by dorkmo** [reply]
wallet will show you the time that is saw your win, not the time it actually happened with the farmer... if your wallet wasnt connected to the node

**Reply by alexs20** [reply]
I have the same problem. Linux version (1.1.2) It is showing 0 connections but everything is in sync. Had to restart the GUI client several times to see it connected again
